hw2/Views/MatchView.swift

```diff
@@ -0,0 +1,140 @@
+    private let awayScoreLabel = UILabel()
+    
+    override func addViews() {
+        super.addViews()
+        addSubview(timeLabel)
+        addSubview(timeStatusLabel)
+        
+        addSubview(dividerImageView)
+        
+        addSubview(homeTeamLogoImageView)
+        addSubview(homeTeamLabel)
+        addSubview(homeScoreLabel)
+        
+        addSubview(awayTeamLogoImageView)
+        addSubview(awayTeamLabel)
+        addSubview(awayScoreLabel)
+    }
+    
+    override func styleViews() {
+        super.styleViews()
+        
+        StylingHelper.styleImageView(imageView: dividerImageView, imageName: "DividerHorizontal")
+        if let regularFont = AppStyles.Fonts.regular(size: 14) {
+            StylingHelper.styleLabel(label: timeLabel, font: regularFont, color: AppStyles.Colors.secondary ?? .gray, textAlignment: .center)
+            StylingHelper.styleLabel(label: homeTeamLabel, font: regularFont, color: AppStyles.Colors.secondary ?? .gray)
+            StylingHelper.styleLabel(label: awayTeamLabel, font: regularFont, color: AppStyles.Colors.primary ?? .black, textAlignment: .left)
+        }
+    }
+    
+    override func setupConstraints() {
+        super.setupConstraints()
+                
+            timeLabel.snp.makeConstraints { make in
+                make.left.equalToSuperview().offset(4)
+                make.top.equalToSuperview().offset(10)
+                make.centerY.equalTo(dividerImageView).offset(-10)
+                make.width.equalTo(56)
+                make.height.equalTo(16)
+            }
+            
+            timeStatusLabel.snp.makeConstraints { make in
+                make.centerX.equalTo(timeLabel)
+                make.top.equalTo(timeLabel.snp.bottom).offset(4)
+                make.width.equalTo(56)
+                make.height.equalTo(16)
+            }
+            
+            dividerImageView.snp.makeConstraints { make in
+                make.centerY.equalTo(homeTeamLogoImageView).offset(10)
+                make.left.equalToSuperview().offset(64)
+                make.width.equalTo(1)
+                make.height.equalTo(40)
+            }
+            
+            homeTeamLogoImageView.snp.makeConstraints { make in
+                make.left.equalTo(dividerImageView.snp.right).offset(16)
+                make.width.height.equalTo(16)
+            }
+            
+            homeTeamLabel.snp.makeConstraints { make in
+                make.centerY.equalTo(homeTeamLogoImageView)
+                make.left.equalTo(homeTeamLogoImageView.snp.right).offset(8)
+                make.width.equalTo(192)
+                make.height.equalTo(16)
+                
+            }
+            
+            homeScoreLabel.snp.makeConstraints { make in
+                make.centerY.equalTo(homeTeamLogoImageView)
+                make.left.equalTo(homeTeamLabel.snp.right).offset(16)
+                make.width.equalTo(32)
+                make.height.equalTo(16)
+            }
```

Comment:
Pošto je ova labela fixna, možeš ju samo zakačaiti desno s trailing i boli ju briga. A onda nek se nameLabel rasteže do nje.